src/Parser/Druid/Balance/Modules/ResourceTracker/AstralPowerTracker.js

```diff
@@ -0,0 +1,17 @@
+import Combatants from 'Parser/Core/Modules/Combatants';
+import RESOURCE_TYPES from 'common/RESOURCE_TYPES';
+
+import ResourceTracker from 'Parser/Rogue/Subtlety/Modules/ResourceTracker/ResourceTracker';
```

Comment:
I take it you're importing Rogue's ResourceTracker because it's an obvious base, but I think it's generally not a good idea to import another spec's module like that. If the Subtlety dev decides to change something he may mess up your module and he'll probably not notice.

Instead, you should make a generalized ResourceTracker in Core somewhere (probably in Modules), and then make both the Subtlety and Balance specs extend it. I imagine in the future there will be other specs that have interest in that as well.